src/components/CreateGameDialog.vue

```diff
@@ -8,16 +8,17 @@
   >
     <template #activator>
       <v-btn
-        class="text-surface-2"
-        color="primary"
+        class="px-16 w-100"
+        color="newPrimary"
+        size="x-large"
+        text-color="white"
         data-cy="create-game-btn"
-        rounded
-        elevation="8"
       >
-        Create Game
+        Create a game
       </v-btn>
     </template>
     <template #body>
+      <h4>Want to play solo? Try <a class="text-cyan-lighten-2 text-decoration-none" href="https://human-ai-interaction.github.io/cuttle-bot/">playing vs AI</a> to learn the ropes and test your mettle.</h4>
```

Comment:
This should open in a new tab
```suggestion
      <h4>Want to play solo? Try <a class="text-cyan-lighten-2 text-decoration-none" href="https://human-ai-interaction.github.io/cuttle-bot/" target="_blank">playing vs AI</a> to learn the ropes and test your mettle.</h4>
```